requirements/production.txt

```diff
@@ -3,7 +3,12 @@
 -r base.txt
 
 gunicorn==20.1.0  # https://github.com/benoitc/gunicorn
+
 psycopg2==2.9.6  # https://github.com/psycopg/psycopg2
+# Keeping psycopg2 for now
```

Comment:
@vincentdavis  we have to keep psycopg2 in base.we need it in local right to connect with postures